src/OpenColorIO/transforms/builtins/ACES.cpp

```diff
@@ -1047,6 +1114,326 @@ void RegisterAll(BuiltinTransformRegistryImpl & registry) noexcept
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-500nit-REC2020_2.0",
+            "Component of ACES 2 Output Transforms for 500 nit HDR Rec2020",
+            500.f,
+            REC2020::primaries,
+            REC2020::primaries,
+            1.f,
+            false
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-1000nit-REC2020_2.0",
+            "Component of ACES 2 Output Transforms for 1000 nit HDR Rec2020",
+            1000.f,
+            REC2020::primaries,
+            REC2020::primaries,
+            1.f,
+            false
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-2000nit-REC2020_2.0",
+            "Component of ACES 2 Output Transforms for 2000 nit HDR Rec2020",
+            2000.f,
+            REC2020::primaries,
+            REC2020::primaries,
+            1.f,
+            false
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-4000nit-REC2020_2.0",
+            "Component of ACES 2 Output Transforms for 4000 nit HDR Rec2020",
+            4000.f,
+            REC2020::primaries,
+            REC2020::primaries,
+            1.f,
+            false
+        },
+        //
+        // D60
+        //
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - SDR-100nit-REC709-D60-in-REC709-D65_2.0",
+            "Component of ACES 2 Output Transforms for 100 nit SDR Rec709 simulating D60 white in Rec709",
+            100.f,
+            REC709_D60::primaries,
+            REC709::primaries,
+            1.f,
+            true
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - SDR-100nit-REC709-D60-in-P3-D65_2.0",
+            "Component of ACES 2 Output Transforms for 100 nit SDR Rec709 simulating D60 white in P3-D65",
+            100.f,
+            REC709_D60::primaries,
+            P3_D65::primaries,
+            1.f,
+            true
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - SDR-100nit-REC709-D60-in-REC2020-D65_2.0",
+            "Component of ACES 2 Output Transforms for 100 nit SDR Rec709 simulating D60 white in Rec2020",
+            100.f,
+            REC709_D60::primaries,
+            REC2020::primaries,
+            1.f,
+            true
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - SDR-100nit-P3-D60-in-P3-D65_2.0",
+            "Component of ACES 2 Output Transforms for 100 nit SDR P3-D60 simulating D60 white in P3-D65",
+            100.f,
+            P3_D60::primaries,
+            P3_D65::primaries,
+            1.f,
+            true
+        },
+        {
+            "ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - SDR-100nit-P3-D60-in-XYZ-E_2.0",
+            "Component of ACES 2 Output Transforms for 100 nit SDR P3-D60 simulating D60 white in XYZ-E",
+            100.f,
+            P3_D60::primaries,
+            CIE_XYZ_ILLUM_E::primaries,
+            1.f,
+            false
```

Comment:
Good points @zachlewis, however I believe it is as intended and following the [aces-output](https://github.com/ampas/aces-output/tree/main/d60/dcdm) reference. My understanding is that the DCDM SDR standard account for a range of whitepoint at peak luminance through the normalization constant so the white scale is not needed (eg. for D60). DCDM HDR however use a straight PQ encoding so would benefit from the white scaling to preserve the whitepoint at peak luminance.

@nick-shaw is that correct?

Reply:
That is indeed correct. That is why the DCDM OTs have a `linear_scale_factor` of 48/52.37 to produce 48 nits on a DCI projector with 52.37 nit peak white for headroom.

Reply:
Thanks Nick. Noting that in the OCIO implementation, the `linear_scale_factor` is applied as part of the Builtins transforms only when the intent was to have peak luminance scaling (eg. for the 108 and 300 nits HDR transforms). When `linear_scale_factor` is used for the DCDM scaling constant, I chose to ignore that and let the OCIO display handle it.

Reply:
Yes. I didn't follow the code path through enough (on my phone) to see how you handled that. But I assumed you did. The DCDM output effectively produces a result normalised to 48 nits, which may have values above 1.0. Then that gets put in a 52.37 nit 'container'.

Reply:
Excellent, that makes perfect sense.

Reply:
I agree that it should be false. And I agree that the 48/52.37 scaling should be left to the OCIO display color space.

However, I'm wondering if the D60 PQ versions should be false as well? They are true in the CTL, so Remi has correctly implemented that, but the reasoning is less clear.

The purpose of the Scale White option seems to be to darken values slightly to avoid clipping when the white moves to be non-equal RGB. This is necessary if the encoding does not go above 1., but PQ has plenty of headroom (much more than DCDM), so it would seem unnecessary.  @nick-shaw , I guess this is done out of a worry about QC rejecting masters that exceed a certain nit level in any channel?

Reply:
I raised that exact question during a meeting regarding the headroom inherent in PQ. The decision was made to scale anyway to keep all channels within range, as you say.

Reply:
By "headroom" I was just referring to the fact that PQ encodes at least up to 10,000 nits, so there is no need to darken the 1000, 2000, or 4000 nit versions to fit. In other words, I'm assuming for the D60 1000 nit version without scaling, that the G channel would be roughly at 1000 and the red channel would be somewhat above that, but would not clip. But I think what Nick is confirming is that QC might fail a master where any channel goes above 1000, even if nothing is clipped and even if the luminance is technically not above 1000, and so that was the argument for darkening it.

Reply:
Thanks for the explanation Doug, agree that it will not clip in the encoding, though I'm not sure about downstream whether there is a [constant] luminance check or if some devices or further conversion steps could simply naïvely clip per channel. Seems good to be on the safe side I guess!

Reply:
> I'm assuming for the D60 1000 nit version without scaling, that the G channel would be roughly at 1000 and the red channel would be somewhat above that, but would not clip.

Correct. If you want the maths for a 1000 nit example:
```
>>> D60_peak_xyY = [0.32168, 0.33767, 1000.0]
>>> D60_peak_XYZ = xyY_to_XYZ(D60_peak_xyY)
>>> vecmul(XYZ_to_P3D65_matrix, D60_peak_XYZ)
array([ 1037.77164583,   996.28782762,   923.30520654])
```
So we end up dividing the values by 1.03777 to keep all channels <= 1000. This does mean the actual luminance of 1000 nit D60 in D65 is 1000 / 1.03777 = ~963.6 nits.